src/crypto/PSAOperationalKeystore.cpp

```diff
@@ -63,6 +65,10 @@ CHIP_ERROR PSAOperationalKeystore::PersistentP256Keypair::Generate()
 
     Destroy();
 
+#ifdef CONFIG_NET_L2_OPENTHREAD
+    chip::DeviceLayer::ThreadStackMgr().LockThreadStack();
```

Comment:
Worth adding a comment that it's a temporary hack.

Reply:
Added :)